specs/playerPageTCs.js

```diff
@@ -0,0 +1,15 @@
+describe('Player Page Tests', ()=> {
+
+    // let searchField = $('[placeholder="Enter search keywords"]')
+    let playerFullName = global.importedJSON.playerFullName()
```

Comment:
It's not a function - it should be the obect's property - use it without () like this:
`global.importedJSON.playerFullName`